packages/graphql-language-service-types/package.json

```diff
@@ -14,21 +14,23 @@
   },
   "license": "MIT",
   "files": [
-    "dist"
+    "dist",
+    "esm"
   ],
   "keywords": [
     "graphql"
   ],
   "main": "dist/index.js",
-  "typings": "src/index",
+  "module": "esm/index.js",
+  "typings": "src/index.ts",
   "scripts": {
     "build": "yarn run build-ts && yarn run build-flow",
     "build-ts": "tsc",
     "build-flow": "node ../../resources/buildFlow.js",
     "prepublish": "node ../../resources/prepublish.js"
   },
   "peerDependencies": {
-    "graphql": "^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0"
```

Comment:
oops, need to fix this. was going change this to ranges in another PR, definately not the move

Reply:
ok resolved, better peer version ranges now and all the way up the PR chain from here :D